src/Microsoft.Health.Dicom.Core/Features/CustomTag/IndexTag.cs

```diff
@@ -13,15 +14,15 @@ namespace Microsoft.Health.Dicom.Core.Features.CustomTag
     /// </summary>
     public class IndexTag
     {
-        public IndexTag(DicomTag tag, DicomVR vr, CustomTagLevel level, bool isCustomTag)
+        public IndexTag(DicomTag tag, DicomVR vr, CustomTagLevel level, CustomTagStoreEntry customTagStoreEntry)
```

Comment:
>public [](start = 8, length = 6)

Why does this need to public? We have public static constructors right?

Reply:
probably could make it private, let me check.

Reply:
changed to private